src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,36 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int TWELVE = 12;
```

Comment:
Stałe określają maksymalny rozmiar tablic, można zastąpić to jedną stała np. MAX_SIZE